### PR TITLE
using Tensile v3.2.0 sgemm assembly kernels for gfx803 and gfx900

### PR DESCRIPTION
**Pull request by guacamoleo**
uses Tensile tag v3.2.0
updated gfx803 and gfx900 .yaml files for sgemm
both sets of files are from benchmarking on Fiji Nano; Vega expected to behave same for assembly kernels since gpus are same size.

### CODE REVIEW

**Comment by guacamoleo** [comment]
in fact, using the fiji benchmark yaml may be more accurate since when benchmarking on Vega the clocks bounce all around.

**Comment by guacamoleo** [comment]
@kknox @amcamd 
This passed rocBLAS validation, can y'all do a code review and approve it?

**Comment by amcamd** [comment]
I have reviewed and approved

**Comment by kknox** [comment]
Did you not want to address the suggestions?  They were minor, but I thought valid

File: install.sh [hunk 1]
```diff
@@ -68,7 +70,7 @@ build_cuda=false
 # check if we have a modern version of getopt that can handle whitespace and long parameters
 getopt -T
 if [[ $? -eq 4 ]]; then
-  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies --options hicd -- "$@")
+  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies --options hicdg -- "$@")
```
[the remark this answers is not in the document]
Comment (guacamoleo) [reply install.sh]: yes, this needs to be fixed

**Pull request merged**